pkgs/misc/scrcpy/default.nix

```diff
@@ -22,8 +22,14 @@ stdenv.mkDerivation rec {
     owner = "Genymobile";
     repo = "scrcpy";
     rev = "v${version}";
-    sha256 = "02szi8w3w0lacyz42hlayxififi863qpm63yg9qir3jcl2vs7vdk";
+    sha256 = "015j1h161c8s2vyh5rbpj0q473zbp5sygw5y7cyv59237d703kcb";
   };
+  postPatch = ''
+    substituteInPlace app/src/screen.c \
+      --replace "SDL_RENDERER_ACCELERATED" "SDL_RENDERER_ACCELERATED || SDL_RENDERER_SOFTWARE"
+    substituteInPlace app/src/command.c \
+      --replace 'const char *const adb_cmd[] = {"shell", "rm", path};' 'const char *const adb_cmd[] = {"shell", "rm", "-f", path};'
```

Comment:
Why is this necessary for nixpkgs and not by default?

Reply:
When copying over the prebuilt binary, it also copies the permissions of the nix store, and thus it cannot delete normally.